modules/rosa-edit-objects.adoc

```diff
@@ -79,8 +79,12 @@ $ rosa edit cluster --cluster=mycluster --interactive
 [id="rosa-edit-ingress_{context}"]
 == edit ingress
 
-Edits the additional non-default application router for a cluster.
+Edits the default application router for a cluster.
 
+[NOTE]
+====
+For information about editing non-default application routers, see _Additional resources_.
```

Comment:
Nit:
```suggestion
For information about editing non-default application routers, see "Configuring the Ingress Controller".
```

Re: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#lead-in-link-sentence-style-for-non-xref-situations